setup.py

```diff
@@ -32,10 +34,9 @@
 DOWNLOAD_URL = 'https://github.com/nipy/PySurfer'
 VERSION = version
 
-def check_dependencies():
 
-    needed_deps = ["IPython",
```

Comment:
The `pysurfer` command line tool drops you into IPython. Also in general you need IPython to do any interactive work.